.github/workflows/android_arm64.yml

```diff
@@ -27,6 +27,7 @@ concurrency:
 
 jobs:
   Build:
+    timeout-minutes: 15
```

Comment:
Are these numbers based on the collected stats (e.g. max time of the longest successful run + N) or just by heart?

Reply:
it based on max of the 3 latest execution times (exception build steps, I set execution time without cache)